File: demo_main.cpp
#include <iostream>
#include <map>
#include <stdio.h>
#include "auto_codes/game_time_conf.h"

using namespace std;

int main()
{

	std::map<int, std::map<string, string> > my_conf = get_conf();
	printf("%s\n", my_conf[2101]["season_name"].c_str());
	printf("%s\n", my_conf[2102]["release_start"].c_str());
    return 0;
}

File: demo_main.py
import auto_codes.game_time_conf as time_conf
import auto_codes.game_reward_conf as reward_conf
import auto_codes.game_guide_conf as guide_conf

conf_time = time_conf.get_conf()
print(conf_time)

conf_reward = reward_conf.get_conf()
print(conf_reward[21011028])


conf_guide = guide_conf.get_conf()
print(conf_guide[3]["desc"], conf_guide[3]["npc"])

